stream.go

```diff
@@ -89,6 +89,15 @@ func (s *Stream) Metadata() *Dictionary {
 	return newDictionaryFromC(s.c.metadata)
 }
 
+// https://ffmpeg.org/doxygen/7.0/structAVStream.html
+func (s *Stream) SetMetadata(d *Dictionary) {
```

Comment:
Could you add tests at the bottom of [this file](https://github.com/asticode/go-astiav/blob/master/stream_test.go) similar to [this](https://github.com/asticode/go-astiav/blob/master/format_context_test.go#L79)?